src/Sitewards/DBCompare/Test/Worker/Item/EmailTemplateWorkerTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Sitewards\DBCompare\Test\Worker\Item;
+
+use Sitewards\DBCompare\Worker\Item\EmailTemplateWorker;
+
+class EmailTemplateWorkerTest extends \PHPUnit_Framework_TestCase
```

Comment:
My question here is should the test be named the abstract when I call the EmailTemplate object. Or can i test direct on the abstract?